styles.css

```diff
@@ -112,40 +112,282 @@ section {
+  max-width: 50%;
+}
+
+.about_head p {
+  font-weight: 400;
+  font-size: 16px;
+  margin-bottom: 12px;
+  line-height: 24px;
+  color: #344563;
+  width: 100%;
+}
+
+.about_head a {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.about_head ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+}
+
+.about_language {
+  padding: 24px 12px;
+  display: flex;
+  flex-direction: column;
+}
 
-@media screen and (min-width: 768px) {
-  .contact img {
-    display: none;
-  }
-
-  .body {
-    height: 100vh;
-  }
-
-  .desktop-header {
-    display: flex;
-    flex-direction: row;
-    list-style-type: none;
-    gap: 32px;
-    justify-content: space-between;
-    padding: 10px 0 10px 10px;
-    animation: fadeIn 3s ease-in;
-    animation-delay: 2s;
-    animation-iteration-count: 2;
-  }
-
-  .navigation .nav {
-    display: none;
-  }
-
-  .headline {
-    height: 100vh;
-    background-image: url("./image/Header\ bg@2x.svg");
-    background-repeat: no-repeat;
-    background-position-x: 100%;
-    background-position: 100%;
-    background-position: 100% 100%;
-  }
-
- 
+.item {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  min-width: 300px;
+  padding: 0;
+  justify-content: space-between;
+  padding-bottom: 12px;
+}
+
+.about_lang {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+  text-align: center;
+  justify-content: baseline;
+}
+
+.about_lang > li {
+  gap: 12px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: row;
+  margin-top: 12px;
+  padding: 6px;
+  align-items: center;
+}
+
+hr {
+  opacity: 0.5;
+  margin: 12px 0;
+}
+
+.contact {
+  padding-top: 80px;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #6070ff;
+  border-radius: 75px 0 0 0;
+}
+
+.contact_footer p {
+  width: 327px;
+  height: 102px;
+  font-family: "poppins", "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact h1 {
+  align-items: center;
+  font-family: 'poppins', "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  padding-left: 40px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  row-gap: 20px;
+  margin: 70px 0;
+  font-family: 'Poppins', sans-serif;
+  align-items: start;
+  width: 327px;
+}
+
+form input {
+  padding: 10px;
+  height: 40px;
+  width: 100%;
+}
+
+form textarea {
+  padding: 10px;
+  width: 100%;
+  outline: none;
+  font-family: 'Poppins', sans-serif;
+  height: 170px;
+  top: 5140px;
+  left: 21px;
+}
+
+.start {
+  width: 150px;
+  size: 17px;
+  font-weight: 500;
+  padding: 10px;
+  margin-bottom: 40px;
```

Comment:
- [ ] As suggested by the previous reviewer, be sure to include the button effect (like `:hover`, `:active`) and also some `border-radius` to look like what we have in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=PBimssV7iaLZjIVt-0)